extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -270,58 +256,66 @@ export class AmpBysideContent extends AMP.BaseElement {
    * @private
    */
   getOverflowElement_() {
-    const createElement = utils.getElementCreator(this.element.ownerDocument);
-
-    const overflow = createElement('div', 'i-amphtml-byside-content-overflow',
-        createElement('div', 'i-amphtml-byside-content-overflow-content',
-            createElement('i', 'i-amphtml-byside-content-arrow-down')
-        ));
-    overflow.setAttribute('overflow', '');
+    const doc = /** @type {!Document} */ (this.element.ownerDocument);
+    const overflow = createElementWithAttributes(doc, 'div', dict({
+      'class': 'i-amphtml-byside-content-overflow',
+      'overflow': '',
+    }));
+    const overflowContent = createElementWithAttributes(doc, 'div', dict({
+      'class': 'i-amphtml-byside-content-overflow-content',
+    }));
+    const arrow = createElementWithAttributes(doc, 'div', dict({
+      'class': 'i-amphtml-byside-content-arrow-down',
+    }));
+    overflowContent.appendChild(arrow);
+    overflow.appendChild(overflowContent);
 
     return overflow;
   }
 
   /** @return {!Element} @private */
   createBySideLoader_() {
-    const createElement = utils.getElementCreator(this.element.ownerDocument);
-
-    const loadingPlaceholder =
-      createElement('div', 'i-amphtml-byside-content-loading-container',
-          createElement('div', 'i-amphtml-byside-content-loading-animation')
-      );
-
-    return loadingPlaceholder;
+    const doc = /** @type {!Document} */ (this.element.ownerDocument);
+    const loadingContainer = createElementWithAttributes(doc, 'div', dict({
+      'class': 'i-amphtml-byside-content-loading-container',
+    }));
+    const loadingAnimation = createElementWithAttributes(doc, 'div', dict({
+      'class': 'i-amphtml-byside-content-loading-animation',
+    }));
+    loadingContainer.appendChild(loadingAnimation);
+
+    return loadingContainer;
   }
 
   /**
    * Updates the element's dimensions to accommodate the iframe's
    *    requested dimensions.
-   * @param {Object} data
    * @private
    */
-  updateSize_(data) {
-    // Calculate new height of the container to include the padding.
-    // If padding is negative, just use the requested height directly.
+  updateSize_() {
+    const args = Array.prototype.slice.call(arguments);
+    const data = args.length ? args[0] : null;
     if (!data) {
       return;
     }
 
-    let newHeight;
-    const height = parseInt(data['height'], 10);
-    if (!isNaN(height)) {
-      newHeight = Math.max(
-          height + (this.element./*OK*/offsetHeight
+    this.getVsync().mutate(() => {
```

Comment:
This needs to be a measure phase, not the mutate phase. `this.getVsync().measure(() => {})`